src/backend/locale/es_ES/LC_MESSAGES/django.po

```diff
@@ -1312,91 +1385,208 @@ msgstr ""
 msgid "Best regards, see you soon!"
 msgstr ""
 
-#: marsha/markdown/admin.py:17 marsha/markdown/models.py:118
+#: marsha/deposit/admin.py:15
+msgid "deposited file"
+msgstr ""
+
+#: marsha/deposit/admin.py:16
+msgid "deposited files"
+msgstr ""
+
+#: marsha/deposit/admin.py:23
+msgid "FileDepository"
+msgstr ""
+
+#: marsha/deposit/admin.py:61
+msgid "DepositedFile"
+msgstr ""
```

Comment:
weird CamelCase format for translated strings.

Reply:
fixed